src/sagemaker_training/process.py

```diff
@@ -60,6 +61,22 @@ def get_debugger_exception_classes():
     return exception_classes
 
 
+def get_tf_exception_classes():
```

Comment:
What about XlaRuntimeError ? that is not covered here.

Reply:
added XlaRuntimeError exception.